internal/exec/exec.go

```diff
@@ -0,0 +1,127 @@
+package exec
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+type Execute struct {
+	StdOut   string
+	StdErr   string
+	ExitCode int
+}
+
+func isExecutable(path string) bool {
```

Comment:
Do we need to bother checking for this?  All good if its just be thorough

Reply:
removed